src/einsteinpy/symbolic/stress_energy_momentum.py

```diff
@@ -1,5 +1,93 @@
-from einsteinpy.symbolic.tensor import BaseRelativityTensor
+import numpy as np
+
+from einsteinpy.symbolic.constants import Cosmo_Const, G, c
+from einsteinpy.symbolic.einstein import EinsteinTensor
+from einsteinpy.symbolic.tensor import BaseRelativityTensor, _change_config
 
 
 class StressEnergyMomentumTensor(BaseRelativityTensor):
-    pass
+    """
+    Inherits from ~einsteinpy.symbolic.tensor.BaseRelativityTensor
+    Class for defining Stress-Energy Coefficient Tensor
+    """
+
+    def __init__(self, arr, syms, config="ll", parent_metric=None):
+        """
+        Constructor and Initializer
+
+        Parameters
+        ----------
+        arr : ~sympy.tensor.array.dense_ndim_array.ImmutableDenseNDimArray or list
+            Sympy Array or multi-dimensional list containing Sympy Expressions
+        syms : tuple or list
+            Tuple of crucial symbols denoting time-axis, 1st, 2nd, and 3rd axis (t,x1,x2,x3)
+        config : str
+            Configuration of contravariant and covariant indices in tensor. 'u' for upper and 'l' for lower indices. Defaults to 'll'.
+        parent_metric : ~einsteinpy.symbolic.metric.MetricTensor or None
+            Corresponding Metric for the Stress-Energy Coefficient Tensor.
+            Defaults to None.
+
+        Raises
+        ------
+        TypeError
+            Raised when arr is not a list or sympy Array
+        TypeError
+            syms is not a list or tuple
+        ValueError
+            config has more or less than 2 indices
+
+        """
+        super(StressEnergyMomentumTensor, self).__init__(
+            arr=arr, syms=syms, config=config, parent_metric=parent_metric
+        )
+        self._order = 2
+        if not len(config) == self._order:
+            raise ValueError("config should be of length {}".format(self._order))
+
+    @classmethod
+    def from_metric(cls, metric):
+        if metric.config == "ll":
+            t_einstein = EinsteinTensor.from_metric(metric)
+            temp = Cosmo_Const * metric.tensor()
+            stress_tensor = c ** 4 / (8 * np.pi * G) * (t_einstein.tensor() - temp)
```

Comment:
remove `temp`, it doesn't seem like a good variable name!

Reply:
also, do something like this
```py
if metric.config != "ll":
    metric = metric.inv()
# rest of the code follows from here
```

There is no need for `ValueError` on upper configuration metric

Reply:
First, let's do this, then I'll see about the tests